test/libdevcore/rlp.cpp

```diff
@@ -40,8 +40,119 @@ namespace js = json_spirit;
 namespace dev
 {
 	namespace test
-	{
-		static void buildRLP(js::mValue& _v, RLPStream& _rlp)
+	{	
+		void buildRLP(js::mValue& _v, RLPStream& _rlp);
+		void checkRLPAgainstJson(js::mValue& v, RLP& u);
+
+		void doRlpTests(json_spirit::mValue& v, bool _fillin)
+		{
+			for (auto& i: v.get_obj())
+			{
+				js::mObject& o = i.second.get_obj();
+				if (test::Options::get().singleTest && test::Options::get().singleTestName != i.first)
+				{
+					o.clear();
+					continue;
+				}
+
+				std::cout << "  " << i.first << std::endl;
```

Comment:
I remember it was required by the gav style to always type std::  and other namespaces
and avoid using namespace.